e2e/api_test.go

```diff
@@ -399,6 +399,68 @@ func TestE2E_TaskEndpoints_Delete(t *testing.T) {
 	validateServices(t, false, []string{"api"}, resourcesPath)
 }
 
+// TestE2E_HealthEndpoint tests the health endpoint. This
+// runs a Consul server and the CTS binary in daemon mode.
+// GET /v1/health
+func TestE2E_HealthEndpoint(t *testing.T) {
+	setParallelism(t)
+
+	srv := testutils.NewTestConsulServer(t, testutils.TestConsulServerConfig{
+		HTTPSRelPath: "../testutils",
+	})
+	defer srv.Stop()
+	tempDir := fmt.Sprintf("%s%s", tempDirPrefix, "health_endpoint")
+
+	clientTLSConfig := testutils.TLSConfig{
+		ClientCert:     defaultCTSClientCert,
+		ClientKey:      defaultCTSClientKey,
+		CAFile:         defaultCTSCACert,
+		VerifyIncoming: true,
+	}
+
+	cases := []struct {
+		name         string
+		isTLSEnabled bool
```

Comment:
that's a good point, this is the first endpoint that won't have the TLS portion tested by the `command_tls` tests since it won't have an associated command.